code/src/main/webapp/WEB-INF/credentials.json

```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"1080865471187-u1vse3ccv9te949244t9rngma01r226m.apps.googleusercontent.com","project_id":"covid-catchers-fixed-gcp","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"LpbG3OQ9SqSckXfkWikxXFUn","redirect_uris":["https://covid-catchers-fixed-gcp.ue.r.appspot.com"],"javascript_origins":["https://8080-ac896ae1-5f0c-45b5-8dfe-19fa4d3d8699.us-east1.cloudshell.dev","https://covid-catchers-fixed-gcp.ue.r.appspot.com"]}}
```

Comment:
Do we want to not check this in? We should confirm with grady but you usually don't want to check this into ever, ever.

Reply:
yup I will remove it